api/tests/routes/backoffice/venues_test.py

```diff
@@ -598,6 +598,60 @@ def test_get_virtual_venue(self, authenticated_client):
         assert "Lieu" in badges
         assert "Suspendu" not in badges
 
+    @pytest.mark.parametrize(
+        "ff_new_nav_activated,has_new_nav,has_old_nav",
+        [
+            (True, False, False),
+            (True, True, False),
+            (True, True, True),
+            (True, False, True),
+            (False, False, False),
+            (False, True, False),
+            (False, True, True),
+            (False, False, True),
+        ],
+    )
+    def test_get_venue_with_new_nav_badges(
+        self, authenticated_client, venue, ff_new_nav_activated, has_new_nav, has_old_nav
+    ):
+        if has_new_nav:
+            user_with_new_nav = users_factories.ProFactory()
+            offerers_factories.UserOffererFactory(user=user_with_new_nav, offerer=venue.managingOfferer)
+            users_factories.UserProNewNavStateFactory(user=user_with_new_nav, newNavDate=datetime.utcnow())
```

Comment:
juste pour éviter le warning ;)
```suggestion
            users_factories.UserProNewNavStateFactory(user=user_with_new_nav, newNavDate=datetime.now(timezone.utc))
```

Reply:
J'y ai pensé mais la règle pylint n'ayant pas été changée ça m'oblige à mettre un pylint ignore

Reply:
arf… effectivement ça sera à rajouter dans la pr de refactor alors